### PR TITLE
Remove flickering when moving between measurements

### PR DESCRIPTION
**Pull request by simenheg**
Remove flickering on the measurement details page when moving between measurements by avoid re-rendering the whole details pane.

This will also come in handy later when implementing a "not found" state by making it easier to distinguish "not found" from "not finished loading".

### CODE REVIEW

File: src/components/modals/KPIProgressModal.vue [hunk 1]
```diff
@@ -100,6 +99,13 @@ export default {
 
   extends: ProgressModal,
 
+  props: {
+    kpi: {
+      type: Object,
+      required: true,
+    },
+  },
```
Comment (petterhj) [inline src/components/modals/KPIProgressModal.vue:107]: Mangler å mate inn `kpi` i `edit-value-modal` i `WidgetKpiProgressHistory.vue`?

File: src/components/modals/KPIProgressModal.vue [hunk 2]
```diff
@@ -146,12 +151,12 @@ export default {
     formatKPIValue,
 
     async onDateSelected() {
-      if (!this.activeKpi) {
+      if (!this.kpi) {
         return;
       }
```
Comment (petterhj) [inline src/components/modals/KPIProgressModal.vue:156]: Er denne sjekken strengt talt nødvendig nå som `kpi` er `required`? 🤔

File: src/components/modals/KPIProgressModal.vue (hunk 1 above)
Comment (simenheg) [reply src/components/modals/KPIProgressModal.vue:107]: Jepp.

**Review by petterhj — APPROVED** [review]
LVGTMWSMC 👍

File: src/components/modals/KPIProgressModal.vue (hunk 2 above)
Comment (simenheg) [reply src/components/modals/KPIProgressModal.vue:156]: > Er denne sjekken strengt talt nødvendig nå som `kpi` er `required`? thinking

Niks, tror ikke det.

**Pull request merged**